staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode.go

```diff
@@ -100,6 +104,12 @@ var Decode cbor.DecMode = func() cbor.DecMode {
 		// Reject the arbitrary-precision integer tags because they can't be faithfully
 		// roundtripped through the allowable Unstructured types.
 		BignumTag: cbor.BignumTagForbidden,
+
+		// When unmarshaling a byte string into a []byte, assume that the byte string
```

Comment:
if this is a hint when decoding strings into []byte, should this comment be changed to:
```suggestion
		// When unmarshaling a string into a []byte, assume that the string
```

Reply:
nevermind, I was confusing "byte string" with "byte slice"